ios/RCTVideo.m

```diff
@@ -516,8 +507,6 @@ - (void)playerItemForSourceUsingCache:(NSString *)uri assetOptions:(NSDictionary
         AVURLAsset *asset = [[AVURLAsset alloc] initWithURL:[components URL] options:options];
         [asset.resourceLoader setDelegate:resourceLoaderDelegate queue:dispatch_get_main_queue()];
         */
-
-        handler([AVPlayerItem playerItemWithAsset:asset]);
```

Comment:
Is there no more need to call the handler in this situation?

Reply:
Nope, the handler is being passed down in line 501

`[self playerItemPrepareText:asset assetOptions:options withCallback:handler];z

Reply:
Check. Awesome!